### PR TITLE
[DOCS-3901] Add Logs Cost Attribution doc

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

Adds Logs Cost Attribution doc with images.

### Motivation

PM request. DOCS-3901.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/faq/logs_cost_attribution.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+### Add a `team` tag
+
+Datadog recommends that you use one of these [tagging methods][5] to add the `team` tag to logs **before ingestion**.
+
+However, if it is necessary to configure the tag during ingestion, follow these steps to create and add a `team` tag.
+
+1. [Create a new `team` attribute](#create-a-new-team-attribute).
+2. [Create a remapper to convert the `team` attribute to a tag](#create-a-remapper-to-convert-the-team-attribute-to-a-tag).
+
+You can use this process to create the attributes you want for breaking down your log usage (for example, by departments, products, regions, and so forth).
```
[the remark this answers is not in the document]
Comment (maycmlee) [reply content/en/logs/faq/logs_cost_attribution.md:52]: Lol yeah I thought I'd change it up here. Is it better to keep it consistent with "so on"?

File: content/en/logs/faq/logs_cost_attribution.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+Then, in the **Populate category** section:  
+      a. Enter `service:a` and `env:prod` in the **All events that match** field.  
+      b. Enter `service_a` in the **Appear under the value name** field.  
+      c. Click **Add**
+6. Add the other teams as separate categories.
+7.  Click **Create**.
+
+#### Create a remapper to convert the `team` attribute to a tag
+
+1. Navigate to the pipeline, click **Add processor**.
```
[the remark this answers is not in the document]
Comment (maycmlee) [reply content/en/logs/faq/logs_cost_attribution.md:71]: So the user is brought to the Logs Pipelines page after the last processor has been set up, and the pipeline should be expanded: https://a.cl.ly/kpuO1G2e

Would it be better to say something like, "In the pipeline" instead?

File: content/en/logs/faq/logs_cost_attribution.md [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+5. In the **Set target attribute or tag key** section, select **Tag key** and enter `team`.
+6. Disable **Preserve source attribute** to make sure the attribute is removed and only the tag is kept.
+7. Enable **Override on conflict**.
+8. Click **Create**.
+
+{{< img src="logs/faq/logs_cost_attribution/team_remapper.png" alt="The create remapper form showing all the data filled in to create a team remapper" style="width:75%" >}}
+
+## Configure custom tags
+
+Create custom tags so you can break down custom log usage metrics into categories that are relevant to your use case. For this example, create the following tags:
```
Comment (maycmlee) [inline content/en/logs/faq/logs_cost_attribution.md:84]:
```suggestion
Create custom tags so you can organize custom log usage metrics into categories that are relevant to your use case. For this example, create the following tags:
```

**Review by maycmlee — COMMENTED** [review]
Thanks @drichards-87 for the review! I committed the suggestions and responded to your questions. I also had a couple of questions.

File: content/en/logs/faq/logs_cost_attribution.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (drichards-87) [reply content/en/logs/faq/logs_cost_attribution.md:71]: Ah, okay, that makes sense. I think "Navigate to the pipeline" is fine.

File: content/en/logs/faq/logs_cost_attribution.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (drichards-87) [reply content/en/logs/faq/logs_cost_attribution.md:52]: It's a minor thing to be honest. For consistency, sure, but it's not one of those instances where I think we need to worry about it. Inconsistencies with product names, terms, and feature names, etc. are the ones we should watch out for. "So forth" works just as well as "so on".

File: content/en/logs/faq/logs_cost_attribution.md [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+1. Navigate to [Logs Pipelines][4].
+2. Click **Add a new pipeline**
+3. Enter the filter for logs you want to attribute costs.
+4. Enter a name for the pipeline. For example, `Cost attribution by team`.
+5. Optionally, add tags and a description.
+6. Click **Create**.
+
+Leave the new pipeline at the end of the list of pipelines. This lets the logs go through the other pipelines so that those tags and attributes are created first.
+
+Add all processors created for this cost attribution example to this new pipeline.
```
[the remark this answers is not in the document]
Comment (drichards-87) [reply content/en/logs/faq/logs_cost_attribution.md:41]: Ah, okay, now I understand. I like the reworded sentence.

**Review by drichards-87 — APPROVED** [review]

**Comment by drichards-87** [comment]
@maycmlee I responded to your questions. Everything looked good to me so I went and approved the PR.

File: content/en/logs/faq/logs_cost_attribution.md [hunk 5]
```diff
@@ -0,0 +1,368 @@
...
+1. Navigate to [Logs Pipelines][4].
+2. Click **Add a new pipeline**.
+3. Enter the filter for logs you want to attribute costs.
+4. Enter a name for the pipeline. For example, `Cost attribution by team`.
+5. Optionally, add tags and a description.
+6. Click **Create**.
+
+Leave the new pipeline at the end of the list of pipelines. This lets the logs go through the other pipelines so that those tags and attributes are created first.
+
+Add all processors created for this cost attribution example to this new pipeline.
```
Comment (maycmlee) [inline content/en/logs/faq/logs_cost_attribution.md:41]:
```suggestion
Add all processors you create for this cost attribution example to the new pipeline.
```

**Pull request merged**